src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/ManageExternalAppointmentsService.kt

```diff
@@ -67,15 +67,19 @@ class ManageExternalAppointmentsService(
     )
   }
 
-  private fun PrisonAppointment.extraInformation() =
-    if (videoBooking.isCourtBooking()) "Video booking for court ${videoBooking.court?.description}" else "Video booking for probation team ${videoBooking.probationTeam?.description}"
+  private fun PrisonAppointment.detailedComments() =
+    if (videoBooking.isCourtBooking()) {
+      "Video booking for a ${videoBooking.hearingType?.lowercase()} court hearing at ${videoBooking.court?.description}\n\n$comments"
```

Comment:
Changed on the back of your comments yesterday @timharrison-moj

Reply:
I wonder actually if I should push this into the Entity.  It has all it needs I think.

Reply:
This is the type of thing I meant :-)

Reply:
Could still be improved though on the back of our Slack conversation.